src/ssh/sshConfig.ts

```diff
@@ -114,11 +137,20 @@ Include "gitpod/config"
                 throw new WrapError(`Could not read ssh config file at ${configPath}`, e);
             }
         }
-
-        const scopeRegex = new RegExp(`START GITPOD INTEGRATION.+Include "gitpod/config".+END GITPOD INTEGRATION`, 's');
-        if (!scopeRegex.test(content)) {
+        const oldContent = content;
+        const scopeRegex = new RegExp(`## START GITPOD INTEGRATION.+END GITPOD INTEGRATION`, 'sg');
+        const matchResult = content.match(scopeRegex);
+        if (matchResult) {
+            const matchContent = matchResult[0];
+            if (matchContent !== gitpodIncludeSection) {
+                content = content.replace(scopeRegex, gitpodIncludeSection);
+                // try to check and delete old file
+                tryDeleteOldMatch(matchContent, gitpodHeader);
```

Comment:
Could it be that we will delete by mistake some real user files?

Reply:
I have add content check inside this function

(check if it contains header `### This file is managed by Gitpod. Any manual changes will be lost.`